openmdao/solvers/test/test_nl_gauss_seidel.py

```diff
@@ -73,6 +74,41 @@ def test_sellar_group(self):
         assert_rel_error(self, prob['y1'], 25.58830273, .00001)
         assert_rel_error(self, prob['y2'], 12.05848819, .00001)
 
+    def test_run_apply(self):
```

Comment:
Is there a test somewhere to count the number of times apply is called so that we can tell if the run_apply flag is working the way we expect?